src/lib/viewers/media/MediaControls.js

```diff
@@ -67,6 +68,8 @@ const FILMSTRIP_FRAME_HEIGHT = 90;
         this.setupSettings();
         this.setupScrubbers();
         this.attachEventHandlers();
+
+        this.hasTouch = Browser.hasTouch();
```

Comment:
Also, can you double check and see if you can use this in a declaration above the constructor? I'm curious of when that code executes, then you can remove it from the constructor entirely